redash/models.py

```diff
@@ -685,6 +688,9 @@ def should_schedule_next(previous_iteration, now, schedule):
             previous_iteration = normalized_previous_iteration - datetime.timedelta(days=1)
 
         next_iteration = (previous_iteration + datetime.timedelta(days=1)).replace(hour=hour, minute=minute)
+    else:
```

Comment:
I thought about it. The issue with that is: 
1) we would need a migrator to migrate legacy queries, many things could go potentially wrong here.
2) we would need to be able to parse cron in the frontend and understand what a cron query mean e.g. -> */5 * * * * is "Every 5 minutes". This is might not be always very straightforward.